api/axis.py

```diff
@@ -1183,20 +1209,24 @@ class ResponseParser(Axis360Parser):
         (3113, "Title ID is not available for checkout") : NoAvailableCopies,
         3114 : PatronLoanLimitReached,
         3115 : LibraryInvalidInputException, # Missing DRM format
+        3116 : LibraryInvalidInputException, # No patron session ID provided -- we don't use this
```

Comment:
These are previously unseen error codes defined in the documentation for the new endpoint.